CMakeLists.txt

```diff
@@ -5,6 +5,18 @@ set(CMAKE_CXX_EXTENSIONS off)
 set(CMAKE_CXX_STANDARD 23)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+find_package(glslang CONFIG)
+find_package(nlohmann_json CONFIG)
+find_package(spirv_cross_core CONFIG)
+if(spirv_cross_core_FOUND)
+    find_package(spirv_cross_glsl REQUIRED)
+endif()
+find_package(SPIRV-Headers CONFIG)
+find_package(SPIRV-Tools-opt CONFIG)
```

Comment:
We will have fork of SPIRV-Tools with custom patches

Reply:
Those patches cannot be made on the upstream repo? :p

Reply:
It doesn't matter, even after merge to upstream will need to wait almost year until patches available for most distros. But yes, I will make pull request to SPIRV-Tools

Reply:
Again, the new functionality can be tested and the system library rejected. The submodule will then be used. Bleeding edge distros should not be punished.

Reply:
Sounds good, let's see how it will work